translations/en-us.yml

```diff
@@ -2574,3 +2575,22 @@ dev-tools:
         show-url-bar: 'Show URL bar -- navigate by URL without reloading'
     features:
         title: 'Features'
+search_help:
```

Comment:
We had decided on snake case for the translations files, but I still see PRs go through with kebob and pascal casing. Having a unified vision here would help.